### PR TITLE
docs: modify README

### PR DESCRIPTION
**Pull request by AgwaB**

### CODE REVIEW

File: docs/TRANSACTION_TRACKER.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+### How to handle error?
+
+If the Henesis server got some errors, (e.g. Unstable network) it returns error with `error` event.
+
+```typescript
+subscription.on('error', error => {
+  /* your error handling code */
+});
+```
```
[the remark this answers is not in the document]
Comment (AgwaB) [reply docs/TRANSACTION_TRACKER.md]: I forgot. it will be added next release.

File: docs/EVENT_SUBSCRIBER.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+  {
+    integrationId: 'integrationId',
+    subscriptionId: 'subscriptionId'
+  }
+);
+```
+
+
+
+### Output data
```
Comment (jasonkim-tech) [inline docs/EVENT_SUBSCRIBER.md]: Move this section to the bottom.

File: docs/TRANSACTION_TRACKER.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+
+
+### Output data
+
+Henesis provide tracking response as follows. you can identify transaction status by looking at `message.data.type` and detailed information is in `message.data.result`.
+
+**pending**
+
+```json
```
Comment (jasonkim-tech) [inline docs/TRANSACTION_TRACKER.md]: Fix indentation correctly

File: docs/TRANSACTION_TRACKER.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+      timeout: number, // the argument of 'henesis.trackTransaction' function
+      confirmation: number // the argument of 'henesis.trackTransaction' function
+  	}
+	}
+}
+```
+
+**receipt, confirmation**
+
+```json
```
Comment (jasonkim-tech) [inline docs/TRANSACTION_TRACKER.md]: Fix indentation correctly

File: docs/TRANSACTION_TRACKER.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+
+### Output data
+
+Henesis provide tracking response as follows. you can identify transaction status by looking at `message.data.type` and detailed information is in `message.data.result`.
+
+**pending**
+
+```json
+{
+  ackId: 'ack id',
```
Comment (junbeomlee) [inline docs/TRANSACTION_TRACKER.md]: plz use double quote `""`

**Pull request merged**